lib/matplotlib/backends/backend_gtk.py

```diff
@@ -399,8 +396,6 @@ def expose_event(self, widget, event):
             x, y, w, h = event.area
             self.window.draw_drawable (self.style.fg_gc[self.state],
                                        self._pixmap, x, y, x, y, w, h)
-        if toolbar:
-            toolbar.set_cursor(toolbar._lastCursor)
         return False  # finish event propagation?
```

Comment:
What's happened to GTK?

Reply:
I removed the wait cursor on GTK.  I honestly have zero interest in fixing a Py2-only backend.